boards/doit/index.rst

```diff
@@ -0,0 +1,10 @@
+.. _boards-doit:
```

Comment:
As I mentioned elsewhere, putting this in `others` is a better option and would help to solve the CI issues.